addons/stock/models/stock_orderpoint.py

```diff
@@ -31,6 +39,8 @@ def default_get(self, fields):
     name = fields.Char(
         'Name', copy=False, required=True, readonly=True,
         default=lambda self: self.env['ir.sequence'].next_by_code('stock.orderpoint'))
+    trigger = fields.Selection([
```

Comment:
not sure trigger is the right term? or if you keep it, auto should become 'scheduler'